tezos/new_context/src/lib.rs

```diff
@@ -88,6 +89,15 @@ where
     fn mem(&self, key: &ContextKey) -> Result<bool, ContextError>;
     // mem_tree - check if directory exists
     fn mem_tree(&self, key: &ContextKey) -> bool;
+    fn find_tree(&self, key: &ContextKey) -> Result<Option<Self>, MerkleError>;
```

Comment:
`find_tree` and `add_tree` need to use `WorkingTree` instead of `Self`, because when this trait is implemented for `TezedgeContext`, we don't want to add (or extract) a context, but a tree.